front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -131,10 +130,11 @@ const PacedTrainItem = ({
             key={occurrence.id}
             isSelected={selectedTimeTableItemId === occurrence.id}
             nextOccurrence={occurrences[index + 1]}
+            isValid={!pacedTrain.invalidReason}
           />
         ))}
       </div>
-      {pacedTrain.isValid && (
+      {pacedTrain.isValid && !pacedTrain.invalidReason && (
```

Comment:
```suggestion
      {pacedTrain.isValid && pacedTrain.duration && (
```
you can also remove `!` in `train-duration` span

Reply:
Done